tests/mock_network/utils.rs

```diff
@@ -510,7 +489,6 @@ pub fn add_connected_nodes_until_split(
         | Event::SectionSplit(..) => (),
         event => panic!("Got unexpected event: {:?}", event),
     });
-    clear_relocation_overrides(nodes);
```

Comment:
So I think I missed how this work because we use to set the relocation before doing thing and then reset them, and we do not need to clear anymore?

Reply:
I this this was remnant from the time where relocation was part of bootstrapping. It hasn't been necessary for some time.